### PR TITLE
Add Shipment Item attributes

### PR DESCRIPTION
**Pull request by dja**
Enables attributes on Shipment line items:
- UPC
- Description
- Lot

### CODE REVIEW

File: lib/newgistics/item.rb [hunk 1]
```diff
@@ -5,6 +5,9 @@ class Item
     attribute :id, String
 
     attribute :sku, String
+    attribute :upc, String
```
Comment (manmartinez) [inline lib/newgistics/item.rb]: When are you using this field?

Comment (manmartinez) [reply lib/newgistics/item.rb]: I see, I just couldn't find the `upc` in the response fields listed in the Newgistics documentation, but I see it comes back as part of the response :+1:

**Comment by dja** [comment]
Awesome, thanks! Can you push the update to rubygems (if you haven’t already)?

**Comment by manmartinez** [comment]
@dja yup, we'll make a new release today :)

**Pull request merged**